harvardcards/static/js/modules/deck-view.js

```diff
@@ -254,8 +254,8 @@ function initModule() {
 				button_text.reverse();
 			}
 
-			$reveal_content_el.removeClass(css_cls[0]).addClass(css_cls[1]);
-			$button_el.text(button_text[0]);
+            $('.reveal').text(button_text[0]);
+            $('.reveal_content').removeClass(css_cls[0]).addClass(css_cls[1]);
```

Comment:
Can you explain why these two lines were changed so they are no longer using the explicit references to the reveal content element and button element? This changes the behavior because now it is doing a DOM lookup for *all* reveal content elements and buttons and manipulating their text contents and classes. 

If that's the desired behavior, then it would be more correct to modify the function parameters to pass in these values rather than hard coding the selectors in the function. Either that, or modify the rest of the function so it is consistent.